src/clj/witan/send/model.clj

```diff
@@ -6,18 +6,13 @@
             [witan.send.send :as send]))
 
 (def seed-year 2017)
-(def projection-year 2023)
 (def random-seed 50)
-(def simulations 10)
-(def target-growth 127.2)
-(def target-variance 729.96)
+(def simulations 1000)
 
 (def send-model-workflow
   "Defines each step of the model"
-  [[:initial-population :prepare-send-inputs]
-   [:initial-send-population :prepare-send-inputs]
+  [[:initial-send-population :prepare-send-inputs]
```

Comment:
I think we want to consider if we want to dump all of the witan.workspace stuff or not. Not as a part of this PR though.

Reply:
Don't make me cry